tests/test_fetch.py

```
@@ -48,3 +80,29 @@ def progress(idx: int, total: int, name: str) -> None:
+    def progress(idx: int, total: int, name: str) -> None:
+        progress_calls.append((idx, total, name))
+
+    # Create a mock Runs object and set total to None to simulate API not providing it
+    mock_runs_obj = MockRuns([mock_wandb_run], total=None)
+
+    with patch("dr_wandb.fetch.wandb.Api") as mock_api:
+        mock_api_instance = Mock()
+        mock_api_instance.runs.return_value = mock_runs_obj
+        mock_api.return_value = mock_api_instance
+
+        runs, histories = fetch_project_runs(
```

Ruff (0.14.14)

[warning] Unpacked variable `histories` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)